api/capi/include/nntrainer.h

```diff
@@ -94,6 +94,17 @@ int ml_nnmodel_construct_with_conf(const char *model_conf, ml_nnmodel_h *model);
  */
 int ml_nnmodel_compile(ml_nnmodel_h model);
 
+/**
+ * @brief train the neural network model.
+ * @details Use this function to train neural network model
+ * @since_tizen 6.x
+ * @param[in] model The NNTrainer model handler from the given description.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_INVALID_PARAMETER Invalid Parameter.
```

Comment:
This is not related with this PR; however, these error codes are already defined in Tizen ML API. You will later need to find a way to share them.

Reply:
I'm not sure but if it is defined some header, then it's ok to use it. I will check where they are and use them later.